client/my-sites/migrate/helpers/index.js

```diff
@@ -35,3 +36,49 @@ export function getImportSectionLocation( siteSlug, isJetpack = false ) {
 		? `https://${ siteSlug }/wp-admin/import.php`
 		: `/import/${ siteSlug }/?engine=wordpress`;
 }
+
+export function getImportFlowByURL() {
+	const url = window.location.href;
+	const parsedUrl = new URL( url );
+	const pathSegments = parsedUrl.pathname.split( '/' );
```

Comment:
> can this be hardcoded somehow, and we change this when we get new paths.
> i.e. a dictionary that maps all known pathSegments[2] to some known flow value instead of just returning it directly.

Sure thing! I've added a new flowMapping variable for this case, for these two known flow (`import-hosted-site` and `import-focused`) they will be return as it is. Other than that, it will default to `onboarding-flow`.

Currently only users from `/start` will return as `onboarding-flow`, because the path ends up like `http://calypso.localhost:3000/setup/site-setup/import?siteSlug=${site_slug}`